src/App.tsx

```diff
@@ -37,7 +37,7 @@ const App: React.FC = () => {
   useEagerConnect()
   useFetchPublicData()
   useFetchProfile()
-  useFetchPriceList()
+  useFetchApiPriceList()
```

Comment:
Renaming any of the old 'prices' state (fetched from API) to include `api` so that it's explicit where this data is coming from